gensim/scripts/word2vec2tensor.py

```diff
@@ -67,8 +68,8 @@ def word2vec2tensor(word2vec_model_path, tensor_filename, binary=False):
     outfiletsv = tensor_filename + '_tensor.tsv'
     outfiletsvmeta = tensor_filename + '_metadata.tsv'
 
-    with open(outfiletsv, 'w+') as file_vector:
-        with open(outfiletsvmeta, 'w+') as file_metadata:
+    with smart_open(outfiletsv, 'w') as file_vector:
```

Comment:
must be `wb`

Reply:
also, good idea to make a  code more "flat", like this
```python
with smart_open(...) as file_vector, smart_open(...) as file_metadata:
    ...
```

Reply:
@menshikh-iv  sorry this escaped me! ATM I don't have time to finish it sorry. I have no idea how to add a test and don't have time to research it. My apologies!

Reply:
@FMCumhaill That's sad, but anyway, thanks for the answer, good luck!